controllers/controller/devworkspacerouting/solvers/basic_solver.go

```diff
@@ -67,12 +80,12 @@ func (s *BasicSolver) GetSpecObjects(routing *controllerv1alpha1.DevWorkspaceRou
 	services := getServicesForEndpoints(spec.Endpoints, workspaceMeta)
 	services = append(services, GetDiscoverableServicesForEndpoints(spec.Endpoints, workspaceMeta)...)
 	routingObjects.Services = services
+	routingAnnotations := config.GetGlobalConfig().Routing.Annotations
```

Comment:
Similar to [here](https://github.com/devfile/devworkspace-operator/pull/1070/files#diff-9bf8bfd60b6c1418e0e13699e8c3437689b65c8841a1bd728dd1506ed4866ae9R74), do you mind adding a note?:
`TODO: Use workspace-scoped routing annotatiosn to allow overriding`

Individual workspaces should be allowed to override the global config's routing settings, but this isin't currently done. It's best we make note of where we need to address this for the future.